framework/src/transfers/MultiAppCopyTransfer.C

```diff
@@ -29,19 +30,19 @@ template<>
 InputParameters validParams<MultiAppCopyTransfer>()
 {
   InputParameters params = validParams<MultiAppTransfer>();
-  params.addRequiredParam<AuxVariableName>("variable", "The auxiliary variable to store the transferred values in.");
-  params.addRequiredParam<VariableName>("source_variable", "The variable to transfer from.");
-
+  params.addDeprecatedParam<AuxVariableName>("variable", "The auxiliary variable to store the transferred values in.", "This is being replaced by 'to_variable'.");
+  params.addDeprecatedParam<VariableName>("source_variable", "The variable to transfer from.", "This is replaced by 'from_variable'.");
+  params.addRequiredParam<VariableName>("to_variable", "The variable that the solution is being transfered into.");
+  params.addRequiredParam<VariableName>("from_variable", "The variable to transfer from.");
+  params.addClassDescription("Copies variables (nonlinear and auxiliary) between multiapps that have identical meshes.");
```

Comment:
Good to know. I'm just saying that it may not make sense or might be a source of error in the general case. I'm not preventing this, I just want to make sure there's critical thought on this design.